Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -229,20 +232,31 @@ def build_params_dict(tags: List[str], attribute_type: List[str], limit: int, pa
         'limit': limit,
         'page': page
     }
+    if from_timestamp:
+        params['from'] = from_timestamp
     return params
 
 
-def clean_user_query(query: str) -> Dict[str, Any]:
+def clean_user_query(query: str, limit: int, page: int = 1, from_timestamp: str | None = None) -> Dict[str, Any]:
     """
     Takes the query string created by the user, adds necessary argument and removes unnecessary arguments
     Args:
         query: User's query string
     Returns: Dict which has only needed arguments to be sent to MISP
     """
+    global LIMIT
     try:
         params = json.loads(query)
         params["returnFormat"] = "json"
         params.pop("timestamp", None)
+        if 'page' not in params:
+            params["page"] = page
+        if 'limit' not in params:
+            params["limit"] = limit
+        else:
+            LIMIT = params["limit"]
```

Comment:
```suggestion
        params["limit"] = params.get("limit") or LIMIT
```
also I didn't understand why we set the LIMIT global (note that it's not a best practice to do it, usually globals are meant to be constants. someone else who reads might miss this spot and when he sees later in the code the usage of LIMIT, he will assume it's 2000 (it just happened to me).